invisible_cities/conftest.py

```diff
@@ -101,6 +107,23 @@ def mc_particle_and_hits_data(electron_MCRD_file):
     efile = electron_MCRD_file
     return efile, name, pdg, vi, vf, p, Ep, nhits, X, Y, Z, E, t
 
+@pytest.fixture(scope='session')
+def mc_particle_and_hits_nexus_data(electron_MCRD_nexus_file):
+    X = [0.07783413, 0.07652718, 0.04626106, 0.03018976, 0.02231551]
+    Y = [-0.00466795, -0.02718811, -0.03123637, -0.04843096, -0.04031885]
+    Z = [0.02206664, 0.08338148, 0.05198042, 0.0628169, 0.06507947]
+    E = [0.00328978, 0.00515397, 0.00501425, 0.00506388, 0.00735444]
+    t = [0.00088132, 0.00163837, 0.00221191, 0.00261107, 0.00284019]
+    name = b'e-'
```

Comment:
Does something *demand* that `name` be `bytes` rather than `str`, or is this just an artifact?

Reply:
This is how the name of the particle is saved in the hdf5 file we create as the nexus output. This particular datum is a `char particle_name[20]`. I would prefer very much this to be a `str`, can you point me towards the correct way to do it?

Reply:
@jmbenlloch suggests to use `decode('utf-8')` to convert `bytes` to `str`. @jerenner, if you're still implementing the other changes, could you try this one, too?

Reply:
We've found a problem using decode as it is, since our byte-strings have fixed length and the "empty" characters at the end are pointless hexadecimal ones. Apparently `decode` doesn't work well on them. So, I've found a (very ugly) way, which, nevertheless, works:

`import re`

`re.sub(r'[^\x20-\x7e].*', '', my_byte_string.decode('utf-8','ignore'))`

@jacg, any comment?

Reply:
In the end, @jmbenlloch found a way to write string in nexus that are read as python-string by pytable. Case closed, hurray!